Sources_v3/APIClient/Endpoints/Payloads/ChannelEditDetailPayload.swift

```diff
@@ -43,4 +60,11 @@ struct ChannelEditDetailPayload<ExtraData: ExtraDataTypes>: Encodable {
 
         try extraData.encode(to: encoder)
     }
+    
+    var pathParameters: String {
+        guard let id = id else {
+            return "\(type)"
+        }
+        return "\(type)/\(id)"
+    }
```

Comment:
nit: When we have time, it would be nice to introduce some kind of `APIPathConvertible` protocol these objects conform to. User will return id, channel cid, etc.